requirements/requirements.txt

```diff
@@ -1,7 +1,7 @@
 envisage == 4.6.0
 stevedore == 1.24.0
-chaco == 4.6.1
 futures == 3.1.1
+chaco == 4.7.1
```

Comment:
we don't need it in requirements.txt anymore. requirements is installed by pip. if it's in edm, we already have chaco in the environment from edm.

It's a bit messy right now, but in principle we should have all dependencies in edm, and install nothing from pip (except for local things, such as force_bdss)

Reply:
Ok, removed it from requriements.txt. We could also move six, pyzmq and envisage into edm_requirements as edm has up to date versions of those packages. Stevedore and futures don't seem to be up to date via edm right now though.